internal/server/v1/event/golden/event_collection/DroughtEvent_BFA_202204.json

```diff
@@ -1,128 +1,3 @@
 {
-  "eventCollection": {
```

Comment:
@Spaceenter @chejennifer data is missing here

Reply:
Please do 2 updates in event_collection_test.go:

1. https://github.com/datacommonsorg/mixer/blob/master/internal/server/v1/event/golden/event_collection_test.go#L100 -- Change 2022-04 to 2022-05

2. https://github.com/datacommonsorg/mixer/blob/master/internal/server/v1/event/golden/event_collection_test.go#L68 -- Change FireEvent to WildlandFireEvent

Then re-run update_golden.sh

Reply:
I updated these lines and reran update_golden.sh - are the goldens expected to change?